projects/laji/src/app/shared-modules/download-request/file-download.service.ts

```diff
@@ -21,7 +21,7 @@ export enum FileType {
 @Injectable({providedIn: 'root'})
 export class FileDownloadService {
   fileType: FileType = FileType.standard;
-  format: FileFormat = FileFormat.shp;
+  format: FileFormat = FileFormat.gpkg;
```

Comment:
File download service uses gpkg now that shp was removed?

Reply:
Yeah, gpkg is the only gis download format now